test/javascripts/components/chat-message-test.js

```diff
@@ -116,4 +116,16 @@ module("Discourse Chat | Component | chat-message", function (hooks) {
       "has the correct hidden css class and expand button within"
     );
   });
+
+  test("Message marked as visible", async function (assert) {
+    this.setProperties(generateMessageProps());
+
+    await render(template);
+    await settled();
```

Comment:
As mentioned in the other comment, let's make this `waitFor` to reduce the chance of flakiness